core/src/main/java/org/apache/brooklyn/core/mgmt/rebind/PeriodicDeltaChangeListener.java

```diff
@@ -382,6 +401,8 @@ protected void persistNowInternal(boolean alreadyHasMutex) {
             if (!alreadyHasMutex) persistingMutex.acquire();
             if (!isActive() && state != ListenerState.STOPPING) return;
             
+            updatePlaneIdIfTimedOut();
```

Comment:
Writes to the datastore are lossy. We'll just log failures and move on. (Most) entities will get updated multiple times in their lifecycle so not a huge deal. planeId does not get updated so if the first write fails it's not available to the HA cluster at all. That's why I decided to periodically write it to the datastore. Will add a comment.